tests/migrations/test_operations.py

```diff
@@ -1447,6 +1452,44 @@ def test_remove_index(self):
         self.assertEqual(definition[1], [])
         self.assertEqual(definition[2], {'model_name': "Pony", 'name': "pony_test_idx"})
 
```

Comment:
This is kind of a separate test case but I have it to this method itself because they share the same initial setup. Hence, added the line break as a demarcation.